tools/package.json

```diff
@@ -24,6 +16,7 @@
     "js-yaml": "^4.1.0",
     "node-fetch": "^3.3.2",
     "tsx": "^4.8.2",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "app-toolbelt": "github:digitalfabrik/app-toolbelt#semver:0.4.0"
```

Comment:
🔧 I think we can remove all other dependencies here, correct?

Reply:
I think so I will take a look 👍️